src/modules/inbox/InboxModule.tsx

```diff
@@ -34,14 +34,14 @@ const InboxModule = () => {
 
   const dispatch = useDispatch();
   const { account, chainId, library } = useWeb3React();
-  const { epnsReadProvider, epnsCommReadProvider } = useSelector((state) => state.contracts);
+  const { epnsReadProvider, epnsCommReadProvider } = useSelector((state:any) => state.contracts);
```

Comment:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/42746736/212844185-fb789942-7d94-486b-9e2b-e11fc6349221.png">
make the input param like this, to resolve the error

Reply:
pushed all review changes...